docs/bonsai-ui/express-checkout/index.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: Express Checkout
+---
+
+:::caution
+This integration is deprecated and will be removed in a future release. Please use [Bonsai Checkout](/docs/bonsai-ui/checkout) instead.
+:::
```

Comment:
Everything else is in this document is the same as is in current `express-checkout` page